### DISCUSSION TITLE
Difference between Soul Devourer and Soul Siphon?

### DISCUSSION THREAD

**Discussion by paperprincessai** [chronosacaria/MCDungeonsWeapons#251] [Q&A] [answered]
![image](https://github.com/chronosacaria/MCDungeonsWeapons/assets/14954885/4818735e-5507-4498-83ea-b90f402092e6)
Soul Devourer is pretty clear about it being 33.3% per level, but Soul Siphon only says experience 1/2/3....what does this mean?  Does that mean there's a 10% chance of 1/2/3 whole levels? exp orbs?

**Comment by ihymyly** [comment, marked as answer]
Just orbs of singular experience if I'm correct. Soul Devourer is strictly better because it grants you to arguably more than 1-3 orbs (x2 at level III is quite a lot) while not limiting you to just "kills".

If you are using this in a modpack of your own, you could change the values (chance) in the config files to suite your preferences.

**Comment by chronosacaria** [comment]
@kurobara31694 I am extremely sorry! I have no idea how I missed this, but yes, @ihymyly is correct. Essentially, Soul Devourer extends to more than just kills. We use XP as the analogue for souls. The trade off is that Soul Siphon is easier to get than Soul Devourer, but it is, essentially, a complete upgrade from Soul Siphon
